official/transformer/transformer_main.py

```diff
@@ -116,26 +194,28 @@ def get_learning_rate(learning_rate, hidden_size, learning_rate_warmup_steps):
     # The full name includes variable and names scope. In this case, the name
     # is model/get_train_op/learning_rate/learning_rate
     tf.identity(learning_rate, "learning_rate")
-    # Save learning rate value to TensorBoard summary.
-    tf.summary.scalar("learning_rate", learning_rate)
 
     return learning_rate
 
 
 def get_train_op(loss, params):
-  """Generate training operation that updates variables based on loss."""
+  """Generate training op and summary dict."""
```

Comment:
Why make this comment less explanatory? Also, update func name to reflect new duties.